d8content/metadatadisplay_entity_18.json

```diff
@@ -0,0 +1 @@
scription>\r\n\t\t{% endfor %}\r\n     {% else %}\r\n\t<dc:description>{{ data.description|escape('html_attr') }}<\/dc:description>\r\n\t{% endif %} \r\n{% endif %}\r\n{# Subject #}\r\n{# LoD Subjects #}\r\n{% if data.subject_loc|length > 0 %}\r\n\t{% for subject_loc in data.subject_loc %}\r\n     \t<dc:subject>{{ subject_loc.label }}<\/dc:subject>\r\n\t{% endfor %}\r\n{% endif %}\r\n{% if data.subject_lcgft_terms|length > 0 %}\r\n\t{% for subject_lcgft_terms in data.subject_lcgft_terms %}\r\n     \t<dc:subject>{{ subject_lcgft_terms.label }}<\/dc:subject>\r\n\t{% endfor %}\r\n{% endif %}\r\n{# Publisher #}\r\n{% if data.publisher|length > 0 %}\r\n\t<dc:publisher>{{ data.publisher }}<\/dc:publisher>\r\n{% endif %}  \r\n{# Language #}\r\n{% if data.language|length > 0 %}\r\n     {% if data.language is iterable %}\r\n     \t{% for language in data.language %}\r\n       \t\t<dc:language>{{ language }}<\/dc:language>\r\n     \t{% endfor %}\r\n     {% else %}\r\n     \t\t<dc:language>{{ data.language }}<\/dc:language>\r\n     {% endif %} \r\n{% endif %}\r\n{# Type #}\r\n{% if data.type|length > 0 %}\r\n\t<dc:type>{{ data.type }}<\/dc:type>\r\n{% endif %}    \r\n{# Format \r\n<dc:format><\/dc:format>\r\n#}\r\n{# Identifier #}\r\n{% if data.local_identifier|length > 0 %}\r\n \t{% if data.local_identifier is iterable %}\r\n     \t{% for local_identifier in data.local_identifier %}\r\n\t\t\t<dc:identifier>{{ local_identifier|escape('html_attr') }}<\/dc:identifier>\r\n        {% endfor %}\r\n       {% else %}\r\n     \t\t<dc:identifier>{{ data.local_identifier|escape('html_attr') }}<\/dc:identifier>\r\n    {% endif %}    \r\n{% endif %}\r\n{# Source #}\r\n{# Source #}\r\n{% if data.owner|length > 0 %}\r\n\t<dc:source>{{ data.owner }}<\/dc:source>\r\n{% endif %}\r\n{# Relation #}\r\n{% if node.id %}\r\n    {% set nodeurl = url('entity.node.canonical', {'node': node.id}, {'absolute': true}) %}\r\n{% else %}\r\n    {% set nodeurl = '<current>' ~ '\/iiif\/manifest' %}\r\n{% endif%}\r\n {% if attribute(data, 'ismemberof') %}\r\n {% if attribute(data, 'ismemberof') is iterable %}\r\n {% for collection in attribute(data, 'ismemberof')%}\r\n   {% set parentnode = bamboo_load_entity('node', collection) %}\r\n   {% set parentnodeurl = url('entity.node.canonical', {'node': collection}, {'absolute': true}) %}\r\n<dc:relation>{{ parentnode.title.value }}<\/dc:relation>\r\n{% endfor %}\r\n{% else %}\r\n  {% set parentnode = bamboo_load_entity('node', attribute(data, 'ismemberof')) %}\r\n  {% set parentnodeurl = url('entity.node.canonical', {'node': attribute(data, 'ismemberof')}, {'absolute': true}) %}\r\n  <dc:relation>{{ parentnode.title.value }}<\/dc:relation>\r\n  {% endif %}\r\n{% endif %}\r\n{# Rights #}\r\n{% if data.rights|length > 0 %} \r\n  <dc:rights>{{ data.rights }}<\/dc:rights>\r\n{% endif %}\r\n<\/oai_dc:dc>\r\n{% endif %}\r\n<\/metadata>\r\n{% endif %}"},"id":"b5c12175-e0a9-49e9-a698-cbba342667e0","type":"metadatadisplay_entity--metadatadisplay_entity"}}
```

Comment:
@alliomeria I think we might have trouble here with the "revision" data ... pretty sure JSON API will complain.
Could you manually remove 
"revision_created":"2024-03-08T16:55:15+00:00" and "drupal_internal__vid":18 ? Same applies to the other one.
The scripts (export import one) was created before I built revisions and I will have to update it for this release too. Sorry

Reply:
Ok, think all the extra new Revision & drupal_internal_vid keys are removed for these now. Understood about that script needing updates...